webdriver/tests/bidi/script/get_realms/sandbox.py

```diff
@@ -0,0 +1,228 @@
+import pytest
+
+from webdriver.bidi.modules.script import ContextTarget
+
+from ... import recursive_compare
+
+PAGE_ABOUT_BLANK = "about:blank"
+
+
+@pytest.mark.asyncio
+async def test_sandbox(bidi_session, top_context):
+    evaluate_result = await bidi_session.script.evaluate_raw(
+        expression="1 + 2",
+        target=ContextTarget(top_context["context"]),
+        await_promise=False,
+    )
+
+    # Create a sandbox
+    evaluate_in_sandbox_result = await bidi_session.script.evaluate_raw(
+        expression="1 + 2",
+        target=ContextTarget(top_context["context"], "sandbox"),
+        await_promise=False,
+    )
+
+    result = await bidi_session.script.get_realms()
+
+    recursive_compare(
+        [
```

Comment:
NIT: as long as the order is not guaranteed, maybe add a helper method `set_of`?